dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/boogie/SvcompProcedures.java

```diff
@@ -203,4 +205,11 @@ private static void __VERIFIER_nondet_bool(VisitorBoogie visitor, Call_cmdContex
 					.setSourceCodeFile(visitor.sourceCodeFile);		
 	    }
 	}
+
+	private static void __VERIFIER_loop_bound(VisitorBoogie visitor, Call_cmdContext ctx) {
+		int bound = ((IExpr)ctx.call_params().exprs().expr(0).accept(visitor)).reduce().getValueAsInt();
+		visitor.programBuilder.addChild(visitor.threadCount, EventFactory.Svcomp.newLoopBound(bound)
+				.setCLine(visitor.currentLine)
+				.setSourceCodeFile(visitor.sourceCodeFile));
```

Comment:
It might be, but is there any benefit in not having them? Does it interfere with anything?

Reply:
I don't think they really interfere, but this kind of data is used for witnesses (both SV-COMP and graphviz) and I don't think we want to populate any of them with annotations

Reply:
I think `graphviz` only cares about visible events to begin with (not sure about SV-COMP), so that should be a non-issue.
But even if these annotations would appear cause of their `Cline`, I would not remove the `Cline` information. I think it is faulty logic on the side of the `graphviz`/`SV-COMP` to solely argue about `Cline`. I would change their code to skip events with `Tag.ANNOTATION` instead (except possibly function calls).

Reply:
You are right about that. However for consistency with all other added events in this class, I would still skip adding that data. We only add that data for locals with nondet values (which are then used for svcomp witness).

Reply:
Ok, I didn't notice that the other events do not get a Cline there. Then I will remove it.